docs/reference/docs/bulk.asciidoc

```diff
@@ -321,7 +327,8 @@ operation.
 =====
 `_index`::
 (string)
-The index name or alias associated with the operation.
+Name of the index associated with the operation. If the operation targeted a
+data stream, this is the associated backing index.
```

Comment:
Minor: I wonder if it would be more clear to say something along the lines of "the backing index into which the document was written" instead of "associated backing index" since a data stream may have many associated backing indices.

Reply:
Good point. I used this wording here and a modified version for failed operations.